playground/TestShop/BasketService/appsettings.json

```diff
@@ -10,6 +10,9 @@
   "Kestrel": {
     "EndpointDefaults": {
       "Protocols": "Http2"
+    },
+    "Endpoints": {
+      "Foo": { "Url": "http://*:5002" }
```

Comment:
This probably wouldn't go in, as it's just to illustrate the PR. Though we'll need to write proper tests for the new feature.